src/khiva/dimensionality.cpp

```diff
@@ -611,29 +612,83 @@ af::array khiva::dimensionality::SAX(af::array a, int alphabet_size) {
     return result;
 }
 
+struct VisagramPoint{
```

Comment:
I think it is my fault. It should be renamed to Visvalingam.

Reply:
I see, hehehehe